docs/security.asciidoc

```diff
@@ -7,6 +7,16 @@ We recommend using firewall rules to ensure only authorized systems can connect.
 
 There is also the option of setting up SSL to ensure data sent to the APM Server is encrypted.
 
+[[secret-token]]
+[float]
+==== Secret token
+
+You can configure a secret token which is sent with every request from the APM agents to the server.
+This string is used to ensure that only your agents can send data to your APM servers.
+Both the agents and the APM servers have to be configured with the same secret token.
+
+NOTE: The usage of a secret token only provides any security when used in combination with having SSL/TLS configured.
```

Comment:
I would also mention that secret token is only for backend endpoints and sourcemap uploads, as frontend requests can not be authenticated.

Reply:
good call, I'll follow up with that separately.